e2e/taskevents/taskevents.go

```diff
@@ -165,3 +168,29 @@ func (tc *TaskEventsTest) TestTaskEvents_CompletedLeader(f *framework.F) {
 	require.Equal(t, api.TaskTerminated, events[5].Type)
 	require.Equal(t, api.TaskKilled, events[6].Type)
 }
+
+// TestTaskEvents_FailedSibling asserts the proper events are emitted for a
+// task when another task in its task group fails.
+func (tc *TaskEventsTest) TestTaskEvents_FailedSibling(f *framework.F) {
+	t := f.T()
+	alloc, taskState := tc.waitUntilEvents(f, "failed_sibling", 7)
+	events := taskState.Events
+
+	// Just because a sibling failed doesn't mean this task fails. It
+	// should exit cleanly. (same as in v0.8.6)
+	require.Falsef(t, taskState.Failed, "task unexpectedly failed after %d events\n%s",
```

Comment:
Just fyi, a `require.Assertions` is embedded in `framwork.F` so you can do `f.Falsef` here

Reply:
Muscle memory is hard to fight :sweat_smile: